Dockerfile

```diff
@@ -2,37 +2,43 @@ FROM php:5.6-apache
 MAINTAINER Pierre Cheynier <pierre.cheynier@sfr.com>
 
 ENV PHPIPAM_SOURCE https://github.com/phpipam/phpipam/archive/
-ENV PHPIPAM_VERSION 1.2
+ENV PHPIPAM_VERSION 1.3
 ENV WEB_REPO /var/www/html
 
 # Install required deb packages
-RUN apt-get update && \
-	apt-get install -y git php-pear php5-curl php5-mysql php5-json php5-gmp php5-mcrypt php5-ldap libgmp-dev libmcrypt-dev && \
-	rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get -y upgrade && \
+	apt-get install -y php-pear php5-curl php5-mysql php5-json php5-gmp php5-mcrypt php5-ldap php5-gd php-net-socket libgmp-dev libmcrypt-dev libpng12-dev
 
 # Configure apache and required PHP modules 
 RUN docker-php-ext-configure mysqli --with-mysqli=mysqlnd && \
 	docker-php-ext-install mysqli && \
+	docker-php-ext-install gd && \
+	docker-php-ext-install sockets && \
 	docker-php-ext-install pdo_mysql && \
 	docker-php-ext-install gettext && \ 
 	ln -s /usr/include/x86_64-linux-gnu/gmp.h /usr/include/gmp.h && \
 	docker-php-ext-configure gmp --with-gmp=/usr/include/x86_64-linux-gnu && \
 	docker-php-ext-install gmp && \
-    docker-php-ext-install mcrypt && \
+	docker-php-ext-install mcrypt && \
 	echo ". /etc/environment" >> /etc/apache2/envvars && \
 	a2enmod rewrite
 
+RUN apt-get purge -y libgmp-dev libmcrypt-dev libpng12-dev && \
+	rm -rf /var/lib/apt/lists/*
```

Comment:
I understand the intent there, but to me it will require another image layer and so it will have a negative effect on the container size at the end.